### ISSUE TITLE
The packageFiles option does not copy files to the build folder, only to the app folder

### ISSUE THREAD

**Issue by dmitriyyavorskiy** [excelsior-oss/excelsior-jet-maven-plugin#57]
Hi, 

I have tried the packageFiles option for my project 

```
<packageFiles>
	<packageFile>
	<type>folder</type>
	<path>${project.basedir}/config</path>
	<packagePath></packagePath>
	</packageFile>
</packageFiles>
```
everything works but testrun( there is no config folder in the build folder and my app depends on that folder). 

Before i used maven-resource-plugin for copying files to build and app folders, but i think it is better to use excelsior plugin features.

**Comment by pjBooms** [comment]
Hi,

Thank you for the bug report.
`packageFiles` are actually copied during Test Run to build directory.
The problem here is empty value for `<packagePath>` parameter that was not tested properly,
The quick workaround is to use
```xml
<packagePath>.</packagePath>
```
dot (.) value for  `<packagePath>` parameter.

We will fix the issue in the next release.

**Comment by pjBooms** [comment]
Fixed in 1.1.0 release
